admin/services/pupil-census.service.js

```diff
@@ -57,20 +61,24 @@ pupilCensusService.uploadToBlobStorage = async (uploadFile) => {
  * Creates a new pupilCensus record
  * @param {Object} uploadFile
  * @param {Object} blobResult
+ * @param {Object} submissionResult
  * @return {Object}
  */
-pupilCensusService.create = async (uploadFile, blobResult) => {
+pupilCensusService.create = async (uploadFile, blobResult, submissionResult) => {
   let dataInput = []
   const csvName = uploadFile.filename && uploadFile.filename.replace(/\.[^/.]+$/, '')
   const blobFileName = blobResult && blobResult.name
   dataInput.push(csvName, blobFileName)
   dataInput = JSON.stringify(dataInput.join(','))
+  const jobStatusCode = submissionResult.errorOutput ? 'CWR' : 'COM'
```

Comment:
It would be better imo to define these magic strings as constants with more meaningful names.